src/signer/src/main/java/ee/ria/xroad/signer/tokenmanager/module/ModuleConf.java

```diff
@@ -252,8 +254,10 @@ private static void parseSection(String uid, SubnodeConfiguration section) {
             return;
         }
 
+        Set<String> slotIds = new HashSet<>(Arrays.asList(getStringArray(section, SLOT_IDS_PARAM)));
```

Comment:
The slot id's are defined to be unsigned long integers by the PKCS11 spec., see http://docs.oasis-open.org/pkcs11/pkcs11-base/v2.40/os/pkcs11-base-v2.40-os.html#_Toc72656028